### ISSUE TITLE
[Dataset request]: SRO_in_HEAs_2024

### ISSUE THREAD

**Issue by jvita** [colabfit/data-lake#43]
### Name

Josh Vita

### Email

vita1@llnl.gov

### Dataset name

SRO_in_HEAs_2024

### Authors

Yifan Cao, Killian Sheriff, Rodrigo Freitas

### Publication link

https://arxiv.org/abs/2401.06622

### Dataset description

Here we consider various approaches for the construction of training data sets for machine learning potentials (MLPs) for CrCoNi and evaluate their performance in capturing SRO and its effects on materials quantities of relevance for mechanical properties, such as stacking-fault energy and phase stability. It is demonstrated that energy accuracy on test sets often does not correlate with accuracy in capturing material properties, which is fundamental in enabling large-scale atomistic simulations of metallic alloys with high physical fidelity. Based on this analysis we systematically derive design principles for the rational construction of MLPs that capture SRO in the crystal and liquid phases of alloys. 

### Software

None

### Permissions

- [ ] I confirm that I have the necessary permissions to submit this dataset

**Comment by gpwolfe** [comment]
Emailed Y. Cao, as also contact for issue #55
